qutebrowser/misc/guiprocess.py

```diff
@@ -142,12 +142,14 @@ def start(self, cmd, args, mode=None):
             self._proc.start(cmd, args)
         else:
             self._proc.start(cmd, args, mode)
+        self._proc.closeWriteChannel();
 
     def start_detached(self, cmd, args, cwd=None):
         """Convenience wrapper around QProcess::startDetached."""
         log.procs.debug("Starting detached.")
         self._pre_start(cmd, args)
         ok, _pid = self._proc.startDetached(cmd, args, cwd)
+        self._proc.closeWriteChannel();
```

Comment:
Can you test (with `:spawn -d`) if `test_stdinclose.py` works without this as well? Since Qt completely detaches from the process, I'd expect it to close stdin anyways, but I'm not sure.